### DISCUSSION TITLE
Roles orders in managements.yml playbook

### DISCUSSION THREAD

**Discussion by oxedions** [bluebanquise/bluebanquise#433] [General]
Hello ! 🙂 

One of my colleagues proposed a small change.

Currently, in managements.yml playbooks, repositories_server is called before repositories_client, this to be able to boostrap first management the first time the stack is deployed, so just after creating manually the initial os and bluebanquise repositories.

However, when using this playbook to deploy other managements (HA, spreading load, whatever), this role order do not fit anymore: repositories_client should come first.

So he proposed to invert order, client before server, and add an instruction in the documentation, for first management bootstrapping part: play first the playbook with -t repositories_server, then execute it full. Using this, the playbook will fit this bootstrap and other managements deployment.

Feedbacks on this ?

**Comment by btravouillon** [comment]
I would propose a bootstrap playbook with the roles sorted as required to install the first node.

**Comment by andriimua** [comment]
also its possible to add AppStrem and BaseOS repositories before _package █ Install {{ repositories_server_packages_to_install | join(' ') }} packages_ task inside repositories_server role

**Comment by oxedions** [comment]
I think we would need multiple things:

1. Order has to be inverted in the default management playbook.
2. We need to provide a bootstrap playbook (first deployment).
3. It would be nice if we could provide a script or/and role to automate all the manual part (Add manual repositories, setup primary NIC, etc.)
